apps/admin-gui/src/app/shared/components/dialogs/add-edit-notification-dialog/add-edit-notification-dialog.component.ts

```diff
@@ -45,10 +45,16 @@ export class AddEditNotificationDialogComponent implements OnInit {
     if(this.data.voId){
       const vo = {
         id: this.data.voId,
-        beanName: "'Vo"
+        beanName: "Vo"
       };
 
       this.editAuth = this.authResolver.isAuthorized('vo-addMail_ApplicationForm_ApplicationMail_policy', [vo]);
+    } else if (this.data.groupId) {
+      const group = {
+        id: this.data.groupId,
+        beanName: "Group"
+      };
+      this.editAuth = this.authResolver.isAuthorized('group-addMail_ApplicationForm_ApplicationMail_policy', [group]);
     }
```

Comment:
```suggestion
      if (this.data.groupId) {
      this.groupsService.getGroupById(this.data.groupId).subscribe(group => {
        this.editAuth = this.authResolver.isAuthorized('group-addMail_ApplicationForm_ApplicationMail_policy', [group]);
        console.log('edit auth');
        console.log(this.editAuth);
      });
    } else if (this.data.voId) {
      const vo = {
        id: this.data.voId,
        beanName: "Vo"
      };
      this.editAuth = this.authResolver.isAuthorized('vo-addMail_ApplicationForm_ApplicationMail_policy', [vo]);
    }
```
This is suggestion for whole block starting from row 45. The method isAuthorized in this case need whole object group. It needs voId in the group object